src/web100-util.c

```diff
@@ -594,13 +594,13 @@ static void print_10gvar_renamed(const char * old_name,
  * client reachable via the input parameter socket FD.
```

Comment:
Can we update the function comment? I don't know if this change makes it obsolete or if it was already obsolete, but I don't understand what it's saying. Is `TES_MSG` supposed to be `TEST_MSG`? Is the "input parameter" the comment refers to now `ctl->socket`?